quamash/__init__.py

```diff
@@ -247,9 +178,9 @@ class QEventLoop(_baseclass):
 	...     loop.run_until_complete(xplusy(2, 2))
 	"""
 
-	def __init__(self, app=None):
+	def __init__(self, app):
 		self.__timers = []
-		self.__app = app or QApplication.instance()
+		self.__app = app
 		assert self.__app is not None, 'No QApplication has been instantiated'
```

Comment:
I clearer error message would be preferable, which informs the user that this is an intended change. Also perhaps a normal exception, such as NotImplementedError (as it is was legal in previous versions) or just a ValueError.